src/Avalonia.Base/Media/Geometry.cs

```diff
@@ -199,4 +202,32 @@ public static Geometry Combine(Geometry geometry1, RectangleGeometry geometry2,
             return new CombinedGeometry(combineMode, geometry1, geometry2, transform);
         }
     }
+
+    public class GeometryConverter : TypeConverter
```

Comment:
For consistency, can it be renamed to `GeometryTypeConverter` ?

Reply:
Also, can you add a single test for this case? @rabbitism

Reply:
Other than that, I don't think compile time parsing needs to be done in this PR. Maybe in the future, if somebody wants to.